src/app.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+const fs = require('fs');
+
+const copyFile = () => {
+  try {
+    if (process.argv[2] === 'cp') {
+      const [file, copy] = process.argv.slice(3);
```

Comment:
Would be better to remove the destructuring assignment of `process.argv` and pass the source and destination file paths as arguments to the copyFile function.